webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationFeatureForm.java

```diff
@@ -275,8 +248,7 @@ public void onSubmit(AjaxRequestTarget aTarget, Form<?> aForm)
             }
         });
 
-        add(layerSelector = new AnnotationFeatureForm.LayerSelector
-            ("defaultAnnotationLayer", annotationLayers));
+        add(layerSelector = new LayerSelector(this, "defaultAnnotationLayer", annotationLayers));
```

Comment:
Can we make a separate issue for this? I am not familiar with IModel. It also wasn't me who implemented this. Maybe we can fix this an other time.